### ISSUE TITLE
Help Simon

### ISSUE THREAD

**Issue by sschmid** [sschmid/Entitas-CSharp#333]
### Simon wants to write docs, but Simon is very busy 😭 

Do you want to help and contribute to Entitas? Writing docs and FAQs would really help me and also the community. Your contribution is very much appreciated!

# Up for grabs

### Wiki

- Is everything up to date?
- Are features explained well enough?
- Are new features and concepts missing (especially after 0.37.0 and type safety)
- Did you find typos or outdated links?
- Are the images up to date?
- Does the table of content make sense?
- Is the `#madeWithEntitas` section missing your game or app?
- Do you want to restructure the wiki?

### You
- Did you write a blog post about Entitas?
- Do you want to do Video Tutorials?
- Do you want to share your experience / games with us?

Together we make the Wiki great again!

**Comment by beck-daniel** [comment]
Updated the Overview page of the wiki to fit the current terms (context <--> pool basically)

**Comment by c0ffeeartc** [comment]
Wiki page can be edited by others vs FAQ issue edited by author only.
It could be both: FAQ wiki page + issue for discussion

**Comment by sschmid** [comment]
Yes, wiki is free to be edited. Also happy with having the FAQ in the wiki

**Comment by WeslomPo** [comment]
Fix link in "Install Entitas and first steps" to examples page, it led to new empty page before.

**Comment by KumoKairo** [comment]
Just to clarify it here too - I'm making beginner video tutorials on a simple Dice Poker game (the project itself is almost done, so first videos are to be expected by the end of March 2017). I'll post a link to a project repo so you Entitas people can take a look and point to things that can be a bit off

**Comment by Adjuvant** [comment]
Added a rather complicated [example of inter-context communication for networking](https://github.com/sschmid/Entitas-CSharp/wiki/Inter-context-communication-in-Entitas-0.39.0). If something simpler is desirable happy to do fix it.

**Comment by IsaiahKelly** [comment]
I created a new "Tutorials" wiki page to contain all tutorial related content since this was spread all over the place. Also restructured all the main links and renamed "Download and setup" to "Quick Start".

@Adjuvant how exactly is that page linked to other content in the wiki so people can find it?

**Comment by Adjuvant** [comment]
@IsaiahKelly sorted, thanks for prompt! Avoided linking till I did a redraft, but likely to forget that, so now added it to tutorials.

**Comment by aefreedman** [comment]
Commenting here because I'm finding the current wiki structure hard to navigate and it's difficult to find the information I need in a timely manner, or even discover that it doesn't exist. The reorganization into many pages has hidden information between clicks and searches.

The information for getting started with Entitas is good, but once you get beyond that, there's little information. The many questions and issues on the tracker and gitter may be symptomatic of this.

I would reference the [Zenject docs](https://github.com/modesttree/Zenject) as a good example. Zenject is arguably more finicky and confusing to set up than Entitas, but I think the Zenject documentation does a great job of providing quick access to information while maintaining coherent organization -- often just by Ctrl/Cmd + F searching.

**Comment by aefreedman** [comment]
@sschmid Re: the `faq` label. I disagree that using the issue tracker as an faq is a good idea. It requires additional maintenance on your part and is also unfriendly/unintuitive for a developer trying to find an answer.

Discussion on questions is good, but for someone who _doesn't_ know the answer and is asking the question, the comments can be confusing or flat-out incorrect. GitHub also doesn't have the stackoverflow-like feature of floating the "selected answer" to the top.

We'd also end up in the annoying situation where the issue-creator owns the original comment and must now maintain their own FAQ issue.

The advantage to having an old-school FAQ page is that it is either out-of-date or current, and if the original writer disappears, someone else can update it.

**Comment by WoLfulus** [comment]
@aefreedman it's already closed. it'll will probably be in the forum

see https://github.com/sschmid/Entitas-CSharp/issues/332#event-1047019127

**Comment by sschmid** [comment]
@aefreedman agree

**Comment by aefreedman** [comment]
@WoLfulus I would apply my same worries about the issue tracker to using the (non-existent) forum as the FAQ location as well.

**Comment by sschmid** [comment]
I can totally see the FAQ as a part of the wiki.

**Comment by WoLfulus** [comment]
@aefreedman true... 
@sschmid yeah, wiki is probably the most common and intuitive place to search for a FAQ

**Comment by aefreedman** [comment]
Made a catch-all  [FAQ page](https://github.com/sschmid/Entitas-CSharp/wiki/FAQ) on the wiki.

I'll be gathering information there. Feel free to help.

**Comment by Lazersquid** [comment]
Added [Code generation shortcut for Visual Studio](https://github.com/sschmid/Entitas-CSharp/wiki/Code-generation-shortcut-for-Visual-Studio) to the wiki.
